patches/chrome-installer-linux-common-installer.include.patch

```diff
@@ -1,42 +1,42 @@
 diff --git a/chrome/installer/linux/common/installer.include b/chrome/installer/linux/common/installer.include
-index e6bc6fea7e6fb244fd01b05f1f63fb66c216b5e3..abdc0290c5497f93ea71369aaf2a63b030727b1c 100644
+index f73118e19e420f1a59248508102e005adf8d72d2..1861082212188bf3abc0d47dbf091f37ad439386 100644
 --- a/chrome/installer/linux/common/installer.include
 +++ b/chrome/installer/linux/common/installer.include
 @@ -75,6 +75,7 @@ process_template() (
      -e "s#@@PACKAGE@@#${PACKAGE}#g" \
      -e "s#@@PACKAGE_ORIG@@#${PACKAGE_ORIG}#g" \
      -e "s#@@PACKAGE_FILENAME@@#${PACKAGE_FILENAME}#g" \
 +    -e "s#@@PACKAGEANDCHANNEL@@#${PACKAGEANDCHANNEL}#g" \
+     -e "s#@@SNAPNAME@@#${SNAPNAME}#g" \
      -e "s#@@PROGNAME@@#${PROGNAME}#g" \
      -e "s#@@CHANNEL@@#${CHANNEL}#g" \
-     -e "s#@@COMPANY_FULLNAME@@#${COMPANY_FULLNAME}#g" \
-@@ -161,6 +162,25 @@ stage_install_common() {
-     install -m 644 "${BUILDDIR}/ui_resources_100_percent.pak" "${STAGEDIR}/${INSTALLDIR}/"
+@@ -162,6 +163,25 @@ stage_install_common() {
+     install -m 644 "${OUTPUTDIR}/ui_resources_100_percent.pak" "${STAGEDIR}/${INSTALLDIR}/"
    fi
  
 +  # brave_resources.pak, contains customization, shields
```

Comment:
we need to circle back and look at this stuff because this looks really fragile to me

Reply:
just to clarify ^^ circle back after merging